pkgs/development/python-modules/acstore/default.nix

```
@@ -0,0 +1,21 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "acstore";
+
+  version = "20230226";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-MDeWOU1cv6dW8LL2UmdUbSwm0vg4YfxvpTDTb15+/+s=";
+  };
+
```

The upstream repo contains [tests](https://github.com/log2timeline/acstore/tree/main/tests). We should run them if possible,

```suggestion

  pythonImportsCheck = [
    "acstore"
  ];
```